src/utils/textData.js

```diff
@@ -0,0 +1,757 @@
+  "ours",
+  "ourselves",
+  "out",
+  "outside",
+  "over",
+  "overall",
+  "own",
+  "p",
+  "particular",
+  "particularly",
+  "per",
+  "perhaps",
+  "placed",
+  "please",
+  "plus",
+  "possible",
+  "presumably",
+  "probably",
+  "provides",
+  "q",
+  "que",
+  "quite",
+  "qv",
+  "r",
+  "rather",
+  "rd",
+  "re",
+  "really",
+  "reasonably",
+  "regarding",
+  "regardless",
+  "regards",
+  "relatively",
+  "respectively",
+  "right",
+  "s",
+  "said",
+  "same",
+  "saw",
+  "say",
+  "saying",
+  "says",
+  "second",
+  "secondly",
+  "see",
+  "seeing",
+  "seem",
+  "seemed",
+  "seeming",
+  "seems",
+  "seen",
+  "self",
+  "selves",
+  "sensible",
+  "sent",
+  "serious",
+  "seriously",
+  "seven",
+  "several",
+  "shall",
+  "she",
+  "she'd",
+  "she'll",
+  "she's",
+  "should",
+  "shouldn't",
+  "since",
+  "six",
+  "so",
+  "some",
+  "somebody",
+  "somehow",
+  "someone",
+  "something",
+  "sometime",
+  "sometimes",
+  "somewhat",
+  "somewhere",
+  "soon",
+  "sorry",
+  "specified",
+  "specify",
+  "specifying",
+  "still",
+  "sub",
+  "such",
+  "sup",
+  "sure",
+  "t",
+  "take",
+  "taken",
+  "tell",
+  "tends",
+  "th",
+  "than",
+  "thank",
+  "thanks",
+  "thanx",
+  "that",
+  "that's",
+  "thats",
+  "the",
+  "their",
+  "theirs",
+  "them",
+  "themselves",
+  "then",
+  "thence",
+  "there",
+  "there's",
+  "thereafter",
+  "thereby",
+  "therefore",
+  "therein",
+  "theres",
+  "thereupon",
+  "these",
+  "they",
+  "they'd",
+  "they'll",
+  "they're",
+  "they've",
+  "think",
+  "this",
+  "thorough",
+  "thoroughly",
+  "those",
+  "though",
+  "three",
+  "through",
+  "throughout",
+  "thru",
+  "thus",
+  "to",
+  "together",
+  "too",
+  "took",
+  "toward",
+  "towards",
+  "tried",
+  "tries",
+  "truly",
+  "try",
+  "trying",
+  "ts",
+  "twice",
+  "two",
+  "u",
+  "un",
+  "under",
+  "unfortunately",
+  "unless",
+  "unlikely",
+  "until",
+  "unto",
+  "up",
+  "upon",
+  "us",
+  "use",
+  "used",
+  "useful",
+  "uses",
+  "using",
+  "usually",
+  "uucp",
+  "v",
+  "value",
+  "various",
+  "very",
+  "via",
+  "viz",
+  "vs",
+  "w",
+  "want",
+  "wants",
+  "was",
+  "wasn't",
+  "way",
+  "we",
+  "we'd",
+  "we'll",
+  "we're",
+  "we've",
+  "welcome",
+  "well",
+  "went",
+  "were",
+  "weren't",
+  "what",
+  "what's",
+  "whatever",
+  "when",
+  "when's",
+  "whence",
+  "whenever",
+  "where",
+  "where's",
+  "whereafter",
+  "whereas",
+  "whereby",
+  "wherein",
+  "whereupon",
+  "wherever",
+  "whether",
+  "which",
+  "while",
+  "whither",
+  "who",
+  "who's",
+  "whoever",
+  "whole",
+  "whom",
+  "whose",
+  "why",
+  "why's",
+  "will",
+  "willing",
+  "wish",
+  "with",
+  "within",
+  "without",
+  "won't",
+  "wonder",
+  "would",
+  "wouldn't",
+  "x",
+  "y",
+  "yes",
+  "yet",
+  "you",
+  "you'd",
+  "you'll",
+  "you're",
+  "you've",
+  "your",
+  "yours",
+  "yourself",
+  "yourselves",
+  "z",
+  "zero",
+];
+
+const suffixes = {
```

Comment:
이 변수명도 위 처럼 바꾸면 어떨까요?
```suggestion
const SUFFIXES = {
```